easybuild/easyblocks/v/vmd.py

```diff
@@ -148,7 +148,8 @@ def configure_step(self):
         # PTHREADS: enable support for POSIX threads
         # COLVARS: enable support for collective variables (related to NAMD/LAMMPS)
         # NOSILENT: verbose build command
```

Comment:
Perhaps add descriptions to these additional parameters as well.

Reply:
Fixed below.